wercker.yml

```diff
@@ -3,6 +3,8 @@ box:
   username: $DOCKERHUB_USER
   password: $DOCKERHUB_TOKEN
 
+no-response-timeout: 10
```

Comment:
[Default timeout is 5 minutes](https://devcenter.wercker.com/supplementary-information/faqs/how-to-bypass-timeouts/), many tasks approach this.

Reply:
if this make difference please make separate commit or PR on this.

Reply:
Done

Reply:
Done

Reply:
Previous runs (not just from this PR) failed from timeout (`Command timed out after no response`):

https://app.wercker.com/checkstyle/checkstyle/runs/build/61cd6914237ec900089d34e5?step=61cd7613c148e300075cdac3

https://app.wercker.com/checkstyle/checkstyle/runs/build/61cd5dc6237ec900089d2cc3?step=61cd70dddca9a20007914d21

https://app.wercker.com/checkstyle/checkstyle/runs/build/61cd2583237ec900089d0bba?step=61cd25d6c148e300075b94e8

Reply:
Documentation about this change is here: http://devcenter-staging.wercker.com/docs/faq/how-to-bypass-timeouts.html